src/classes/Funk.ts

```diff
@@ -25,9 +26,15 @@ export class Funk {
      */
     constructor(options?: FunkOptions) {
         if (options) this.options = options;
-        this.ws = new WebSocket(this.options.urlOverride || "wss://funk.statio.cc");
-        this.ws.onopen = () => this.onOpen(this);
-        this.ws.onmessage = (data: string | MessageEvent<any>) => this.onMessage(this, data);
+        if (typeof process === "object") {
```

Comment:
This method of checking the environment is not completely reliable. Instead, check if `typeof window === 'undefined'` for a more reliable way to check.

Reply:
That's fixed :)